tests/testthat/test-search-pv.R

```diff
@@ -90,27 +98,27 @@ test_that("Throttled requests are automatically retried", {
   skip_on_cran()
 
   res <- search_pv('{"_gte":{"patent_date":"2007-01-04"}}', per_page = 50)
-  patent_numbers <- res$data$patents$patent_number
+  patent_ids <- res$data$patents$patent_id
```

Comment:
Is this test actually testing what we want it to now? Seems like there's no possible throttling happening now.

Reply:
The API is throttling, I updated the test to prove it.